src/img/Group.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="50" height="63" viewBox="0 0 50 63" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@tsvetelinak0 

**Why uploaded this image but not used it?**

Are you planning on using it in a different pull request? Maybe you could have uploaded it then?

**I recommend using all lowercase for file names.**

Because some systems are CaSESensITive and some are not which means sometimes things work in development but not in production because of capitals in file names. So if you keep everything lowercase it avoids those problems.

Also git used to have an issue with renaming files to the same name with different capitalisation. If I remember correctly the way to do it is to:

1. copy Group.svg to your desktop and rename it to group.svg
2. ```git rm Group.svg```
3. Drag the backup back into the project folder
4. ```git add group.svg```
5. commit the change

Reply:
yes, the small icon will be used in the green section for the venue address

e.g.
<img width="278" alt="Screenshot 2021-02-01 at 13 23 49" src="https://user-images.githubusercontent.com/27692549/106464672-dd8cea00-6490-11eb-851b-ddc478b23f48.png">

Will rename it with lower cases and commit!

maybe should have not raise the PR but just push the changes to that branch until is ready :)

Reply:
The Icon will be added with another ticket :) my bad!

Reply:
> Will rename it with lower cases and commit!

OK, when you have done that let me know and I will approve and merge this PR